modules/cnv-subscribing-to-the-catalog.adoc

```diff
@@ -20,20 +20,13 @@ Operators.
 
 . Read the information about the Operator and click *Install*.
 
-. On the *Create Operator Subscription* page:
```

Comment:
This section that was removed looks accurate. We should leave it. The CNV docs appear to already have been updated.

Reply:
+1. I updated this in https://github.com/openshift/openshift-docs/pull/20118 two weeks ago. Good catch, @spadgett :)

Reply:
Whoops! Thanks! Good catch indeed.